1.0/python/slim/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016, Google Inc.
```

Comment:
Who is "Google Inc."? When is 2016?

Reply:
I copied the license header information from the existing 1.0. I'll change it to "2018 The gRPC Authors"